pkg/version/version.go

```diff
@@ -22,6 +22,8 @@ var (
 	majorFromGit string
 	// minor version
 	minorFromGit string
+	// build date in ISO8601 format, output of $(date -u +'%Y-%m-%dT%H:%M:%SZ')
+	buildDate string
```

Comment:
It's part of upstream code, and I'd rather have it and be consistent.

On Thu, Jul 7, 2016 at 3:40 PM, David Eads <notifications@github.com> wrote:

> In pkg/version/version.go
> <https://github.com/openshift/origin/pull/9578#discussion_r69972792>:
>
> > @@ -22,6 +22,8 @@ var (
> >  	majorFromGit string
> >  	// minor version
> >  	minorFromGit string
> > +	// build date in ISO8601 format, output of $(date -u +'%Y-%m-%dT%H:%M:%SZ')
> > +	buildDate string
>
> Why do we care about the date now?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9578/files/cfedbc1708fc75ac73aba9d0e9e115b906cd8527..e3c71ab7b9d54bf3eb39c984f59d1decbf2b51f5#r69972792>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_pwLoOv_S0kISNgH4Ex60_kvhY1qbks5qTVYbgaJpZM4I-xTE>
> .
>